Consul/Token.cs

```diff
@@ -224,9 +225,26 @@ public Task<WriteResult<TokenEntry>> Clone(string id, CancellationToken ct = def
         /// <param name="writeOptions">Customized write options</param>
         /// <param name="ct">Cancellation token for long poll request. If set, OperationCanceledException will be thrown if the request is cancelled before completing</param>
         /// <returns>A write result containing the created ACL token</returns>
-        public async Task<WriteResult<TokenEntry>> Clone(string id, WriteOptions writeOptions, CancellationToken ct = default(CancellationToken))
+        public Task<WriteResult<TokenEntry>> Clone(string id, WriteOptions writeOptions, CancellationToken ct = default(CancellationToken))
```

Comment:
Has the async been deleted deliberatly?

Reply:
Yes, to be consistent with the way the other methods call their concrete async method.

See for example: https://github.com/G-Research/consuldotnet/blob/4ac6d334b05b27771bcca95b6327d5592d935435/Consul/Token.cs#L163-L185 

The existing `Clone(string id, CancellationToken ct)` method was also like that:
https://github.com/G-Research/consuldotnet/blob/4ac6d334b05b27771bcca95b6327d5592d935435/Consul/Token.cs#L210-L219